Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -6923,6 +6923,8 @@ script:
         name: device_filter_string
         required: false
         secret: false
+      - description: Target number of the firewall. Use only on a Panorama instance.
+        name: target
```

Comment:
```suggestion
      - description: Target number of the firewall. Use only on a Panorama instance.
        name: target
        required: false
```
Add this to all below as well.

Reply:
Should be done, but can you double check?

Reply:
you did great, marking as resolved.